dom-event-delegation/main.js

```diff
@@ -0,0 +1,14 @@
+var taskList = document.querySelector('.task-list');
+taskList.addEventListener('click', taskDone);
+
+function taskDone(event) {
+  console.log('event target:', event.target);
+  console.log('event.target.tagName:', event.target.tagName);
+
+  if (event.target.tagName === 'BUTTON') {
+    var parent = event.target.closest('.task-list-item');
+    console.log('closest .task.list.item:', parent);
+    var item = document.querySelector('.task-list-item');
+    item.remove();
```

Comment:
This will only ever remove the top item in the list, you should use the `parent` variable you made right above this.